app/javascript/locationsearch.js

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
I tried to use the mapbox js library but had problem loading it into the page. either I was not following the proper new rails method or the examples from mapbox js library were outdated. Ended up writing the js code and just using the API endpoint. The code is pretty minimal comparing to the library